Dockerfile.redhat

```diff
@@ -179,6 +183,14 @@ RUN cd /ovms/src/example/SampleCpuExtension/ && make
 
 RUN bazel test ${debug_bazel_flags} --jobs $JOBS --test_summary=detailed --test_output=streamed  //src:ovms_test
 
+COPY check_coverage.bat /ovms/
+# Test Coverage
+RUN if [ "$CHECK_COVERAGE" == "1" ] ; then true ; else exit 0 ; fi ; wget https://github.com/linux-test-project/lcov/releases/download/v1.16/lcov-1.16-1.noarch.rpm &&\
+    yum install -y lcov-1.16-1.noarch.rpm &&\
+    bazel coverage --combined_report=lcov //src:ovms_test &&\
+    genhtml --output genhtml "$(bazel info output_path)/_coverage/_coverage_report.dat" &&\
+    ./check_coverage.bat && exit $?
```

Comment:
I guess it not possible to run coverage check before main building process, so we could fail faster?

Reply:
I can try it.

Reply:
Done.